src/components/charts/combo.js

```diff
@@ -71,11 +71,10 @@ const comboOptions = {
 
 export default class Combo extends Component {
 
-	render() {
-		return(
-			<div>
-				<Chart container={'combo-chart'} options={comboOptions}/>
-			</div>
-		)
-	}
+    render() {
+        return (
+            <Chart container='main-content'
+                   options={comboOptions}/>
```

Comment:
How often do you pull from master?? This file does not exist already ))

Reply:
I've already merged master into my branch and all conflicts was fixed.

Reply:
Have you committed and pushed your changes after solving conflicts? Because it still has all the conflicts:
![image](https://user-images.githubusercontent.com/26466644/32248755-f544086c-be86-11e7-9d0c-e8c5601a39db.png)